microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -11,16 +11,17 @@ ms.date: 02/11/2022
 # Debug WebView2 apps with Visual Studio Code
 
 Use Microsoft Visual Studio Code to debug scripts that run in WebView2 controls.  <!-- Make sure you're using Visual Studio Code version [insert build here] or later. -->
+Visual Studio Code has a built-in [debugger](https://code.visualstudio.com/docs/nodejs/browser-debugging) for browser debugging.
 
 
 <!-- ====================================================================== -->
 ## Create a launch.json file
 
-To debug your code, your project is required to have a `launch.json` file.  If your project doesn't have a `launch.json` file, create a new `launch.json` file and paste the following code into it:
+To debug your code, your project is required to have a `launch.json` file.  There are two request types, `launch` and `attach`. The following code demonstrates launching the app from Visual Studio Code. To do this, the app must have been built previously. If your project doesn't have a `launch.json` file, create a new `launch.json` file in the `.vscode` subfolder in your current project and paste the following code into it:
```

Comment:
```suggestion
To debug your code, your project is required to have a `launch.json` file.  There are two request types, `launch` and `attach`. The following code demonstrates launching the app from Visual Studio Code (rather than attaching the debugger to a running browser instance). To do this, the app must have been built previously. If your project doesn't have a `launch.json` file, create a new `launch.json` file in the `.vscode` subfolder in your current project and paste the following code into it:
```
or separate first part into a paragraph defining the two request types, launch vs. attach.

Reply:
I find that the "there are two request types" part comes a bit out the blue. If I've never used VSCode before (or if I've used it but never came across launch.json files), I don't know what a request is. It's great to say that a launch.json file is needed, but the sentence could use something that says "A launch.json file describes how ... and needs to define a request type that describes ....". And then you can go on to say that there are two types of requests.
Does that make sense?